### DISCUSSION TITLE
Please improve the UI/UX design for the Webpage and Homepage

### DISCUSSION THREAD

**Discussion by 7549brijesh** [benbusby/whoogle-search#460] [Q&A] [answered]
I am Sorry!!! I am not satisfied with User Interface
Can you please Improve a UI a Little bit more because I am love with whoogle but the UI makes me sad
I know there is option for customising the interface via css but I think UI should be clean and awesome out of the box

Forgive me!! If I am expecting too much!

**Comment by alefvanoon** [comment, marked as answer]
Whats your problem?
![image](https://user-images.githubusercontent.com/53198048/137356667-3e7188ba-d188-4bf5-b9b3-02d9538f6b0b.png)
![image](https://user-images.githubusercontent.com/53198048/137356821-11da62c7-6966-4f88-b433-d1a073f5dccb.png)
It's so cute!

**Reply by 7549brijesh** [reply]
Wow it looks awesome!!!!!!! what did you do?

**Comment by Albonycal** [comment]
yea, but I use this  for myself 
![image](https://user-images.githubusercontent.com/67057319/137359101-db587086-7e80-4b64-bb90-888bba1956d4.png)

**Reply by 7549brijesh** [reply]
can you tell me how did you do that?

**Reply by Albonycal** [reply]
I made my own html file :D 
you can also download it ;-;

**Comment by 7549brijesh** [comment]
my is so boring
![2021-10-14_22-30_1](https://user-images.githubusercontent.com/25589680/137365193-63d748fc-d74a-4020-b906-37064f852c40.png)
![2021-10-14_22-30](https://user-images.githubusercontent.com/25589680/137365195-01d0f856-78d6-4629-beb2-315e99f91b78.png)

**Reply by alefvanoon** [reply]
Just use CSS to customize your instance.
This is my instance CSS(https://s.alefvanoon.xyz):
```

:root {
    /* LIGHT THEME COLORS */
    --whoogle-logo: #685e79;
    --whoogle-page-bg: #ffffff;
    --whoogle-element-bg: #4285f4;
    --whoogle-text: #000000;
    --whoogle-contrast-text: #ffffff;
    --whoogle-secondary-text: #70757a;
    --whoogle-result-bg: #ffffff;
    --whoogle-result-title: #1967d2;
    --whoogle-result-url: #0d652d;
    --whoogle-result-visited: #4b11a8;

    /* DARK THEME COLORS */
    --whoogle-dark-logo: #685e79;
    --whoogle-dark-page-bg: #101020;
    --whoogle-dark-element-bg: #0c54cb;
    --whoogle-dark-text: #ffffff;
    --whoogle-dark-contrast-text: #ffffff;
    --whoogle-dark-secondary-text: #bbbbbb;
    --whoogle-dark-result-bg: #212131;
    --whoogle-dark-result-title: #64a7f6;
    --whoogle-dark-result-url: #34a853;
    --whoogle-dark-result-visited: #bbbbff;
}

#whoogle-w {
    fill: #4285f4;
}

#whoogle-h {
    fill: #ea4335;
}

#whoogle-o-1 {
    fill: #fbbc05;
}

#whoogle-o-2 {
    fill: #4285f4;
}

#whoogle-g {
    fill: #34a853;
}

#whoogle-l {
    fill: #ea4335;
}

#whoogle-e {
    fill: #fbbc05;
}
```

**Reply by 7549brijesh** [reply]
thanks for help

**Comment by dokbua** [comment]
Would be nice to have the result on a full width or responsive page instead of narrow column in the middle.
